test/types/reply.test-d.ts

```diff
@@ -70,6 +74,14 @@ interface ReplyHttpCodes {
   }
 }
 
+interface InvalidReplyHttpCodes {
+  Reply: {
+    '1xx': number,
```

Comment:
`1xx` should be a correct http code.
It is a fallback for all status code starting with `1`